geonode/api/resourcebase_api.py

```diff
@@ -410,14 +410,15 @@ def get_search(self, request, **kwargs):
             objects = []
 
         object_list = {
-           "meta": {"limit": settings.CLIENT_RESULTS_LIMIT,
-                    "next": next_page,
-                    "offset": int(getattr(request.GET, 'offset', 0)),
-                    "previous": previous_page,
-                    "total_count": total_count,
-                    "facets": facets,
-                    },
-            'objects': map(lambda x: self.get_haystack_api_fields(x), objects),
+           "meta": {
+                "limit": settings.CLIENT_RESULTS_LIMIT,
```

Comment:
I'm happy to change it to that variable instead - but either way it should be a variable and not a hard coded value.  Just let me know if you want me to change it and I'll update the PR.